matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/common/CommonTestHelper.kt

```diff
@@ -68,7 +68,8 @@ class CommonTestHelper(context: Context) {
                     context,
                     MatrixConfiguration(
                             applicationFlavor = "TestFlavor",
-                            roomDisplayNameFallbackProvider = TestRoomDisplayNameFallbackProvider()
+                            roomDisplayNameFallbackProvider = TestRoomDisplayNameFallbackProvider(),
+                            syncRequestTimeoutMillis = 5_000L
```

Comment:
OOI I am not sure to understand why it should speed up the tests, since if there is something new server side, the server will tell the client immediately, no?